integration_tests/conftest.py

```diff
@@ -1,21 +1,47 @@
 import pytest
 from cloudpathlib import S3Path
 import os
+import boto3
 
+PIPELINE_BUCKET = os.environ.get("INGEST_PIPELINE_BUCKET")
+S3_CLIENT = boto3.client("s3")
 
-def get_bucket_files_with_suffix(bucket: S3Path, suffix: str) -> list[S3Path]:
-    """Get all the files in a bucket with a given suffix."""
-    bucket_files = []
-    for pattern in ["*", "*/*", "*/*/*", "*/*/*/*"]:
-        files = list(bucket.glob(pattern + suffix))
-        bucket_files.extend(set(files))
-    return bucket_files
```

Comment:
So was this what was causing those test failures?

Reply:
Yep for some reason the glob just wasn't working. Switching to boto3 seemed to resolve the issue. 

I don't know whether some check in the poetry deps affected cloud path lib or something.